packages/files/directory.pony

```diff
@@ -1,12 +1,12 @@
 use "time"
 
-use @ponyint_o_rdonly[I32]()
-use @ponyint_o_rdwr[I32]()
-use @ponyint_o_creat[I32]()
-use @ponyint_o_trunc[I32]()
-use @ponyint_o_directory[I32]()
-use @ponyint_o_cloexec[I32]()
-use @ponyint_at_removedir[I32]()
+use @pony_o_rdonly[I32]()
+use @pony_o_rdwr[I32]()
+use @pony_o_creat[I32]()
+use @pony_o_trunc[I32]()
+use @pony_o_directory[I32]()
+use @pony_o_cloexec[I32]()
+use @pony_at_removedir[I32]()
```

Comment:
Since these functions can be called from Pony programs, they have to be exported (with the new PONY_API macro) in order to be visible from the JIT. I thought it would make sense to semantically mark the functions as public when they must be exported.

But maybe we need a new "visibility" level, since public functions basically mean we should guarantee some form of API stability, and having to uphold that guarantee for FFI functions internal to the standard library doesn't seem ideal. Maybe something like:

- `pony_*`: the public API of the runtime, exported and somewhat stable (once we've settled on what "stable" means)
- `ponyffi_*`: the interface between the runtime and the standard library, exported but with no stability guarantee
- `ponyint_*`: stuff that only needs to be used from the runtime, also no stability guarantee

Would that make sense?